src/navigations/KeyboardWrapper.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import {ViewStyle, KeyboardAvoidingView, Platform, ScrollView} from 'react-native';
+import dimen from '../utils/dimen';
+
+type Props = {
+  children: any;
+  keyboardVerticalOffset?: number;
+  contentContainerStyle?: ViewStyle;
+};
+
+const KeyboardWrapper = ({children, keyboardVerticalOffset}: Props): JSX.Element => {
+  return (
+    <KeyboardAvoidingView
+      style={{flex: 1}}
+      keyboardVerticalOffset={keyboardVerticalOffset || dimen.normalizeDimen(40)}
+      {...(Platform.OS === 'ios' ? {behavior: 'padding'} : {})}>
+      {children}
+    </KeyboardAvoidingView>
+  );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `contentContainerStyle` prop is defined but never used in the component. If it's intended to allow customization of the container style, it should be applied to the `KeyboardAvoidingView` component. If it's not needed, it should be removed from the `Props` type definition.
```diff
- const KeyboardWrapper = ({children, keyboardVerticalOffset}: Props): JSX.Element => {
+ const KeyboardWrapper = ({children, keyboardVerticalOffset, contentContainerStyle}: Props): JSX.Element => {
  return (
    <KeyboardAvoidingView
-     style={{flex: 1}}
+     style={[{flex: 1}, contentContainerStyle]}
      keyboardVerticalOffset={keyboardVerticalOffset || dimen.normalizeDimen(40)}
      {...(Platform.OS === 'ios' ? {behavior: 'padding'} : {})}>
      {children}
    </KeyboardAvoidingView>
  );
};
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->